### DISCUSSION TITLE
What command can I enter where to automatically delete or flush queued messages?

### DISCUSSION THREAD

**Discussion by bukunmilab** [postalserver/postal#3172] [Help with using Postal]
Hey guys, 
What command can I enter in SSH or postal.yml file or database to automatically flush or delete all queued messaged after x hours?

**Comment by willpower232** [comment]
Wouldn't you want to wait for the queue to send all the emails even if it takes hours?

**Comment by bukunmilab** [comment]
Sending suddenly dropped from 400 emails per minute to 83 emails per minute. Too bad. 
But each time I clear the cache the it goes up to 200-300 per minute again. Something must be wrong.

**Reply by willpower232** [reply]
interesting, sounds more like a missing index or query issue on the database table then

**Comment by bukunmilab** [comment]
Can you suggest how to fix this. 
I have no idea what index  might be missing or query to fix. 

Please assist. I can send my credentials privately to have a look.
